fluent-package/apt/systemd-test/downgrade-to-v4.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -exu
+
+. $(dirname $0)/../commonvar.sh
+
+# Install v4
+sudo apt install -y curl ca-certificates
+curl -fsSL https://toolbelt.treasuredata.com/sh/install-${distribution}-${code_name}-td-agent4.sh | sh
+
+# td-agent is running
+systemctl status --no-pager td-agent
+
+# fluent-package 5 (LTS)
+#curl -fsSL https://toolbelt.treasuredata.com/sh/install-${distribution}-${code_name}-fluent-package5-lts.sh | sh
+#sudo apt purge -y fluent-lts-apt-source
+
+# Ensure to install the current
+sudo apt install -V -y \
+    /host/${distribution}/pool/${code_name}/${channel}/*/*/fluent-package_*_${architecture}.deb
+
+# td-agent.service is already masked (link to /dev/null), and remove td-agent.service alias not to conflict with v4
+sudo systemctl unmask td-agent
+
+# Keep log and .conf and so on. dpkg reports "rc fluent-package" and "rc td-agent" status.
+sudo apt remove -y fluent-package td-agent
```

Comment:
Do we need to specify `td-agent`?
Wouldn't it be automatically removed?

```suggestion
sudo apt remove -y fluent-package
```

Reply:
No need to specify it. Thanks.